src/UI/admin/components/available-hall/AdminAvailableHall.js

```diff
@@ -0,0 +1,254 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+const Wrapper = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  width: 374px;
+  height: 400px;
+  border-radius: 4px;
+  border: solid 1px #229c7f;
+  background-color: #ffffff;
+`;
+
+const Header = styled.div`
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  width: 374px;
+  height: 40px;
+  border-radius: 3px;
+  background-color: #229c7f;
+`;
+
+const Title = styled.h1`
+  width: 374px;
+  height: 24px;
+  font-family: Poppins;
+  font-size: 16px;
+  font-weight: 500;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1;
+  letter-spacing: normal;
+  color: #ffffff;
+`;
+
+const Image = styled.img`
+  width: 244px;
+  height: 136px;
+  margin-bottom: 14px;
+  border-radius: 2px;
+  border: solid 1px #229c7f;
+`;
+
+const MainPart = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  width: 372px;
+  height: 360px;
+  padding: 16px 32px 16px 32px;
+  background-color: #ffffff;
+`;
+
+const TextPart = styled.div`
+  box-sizing: border-box;
+  width: 310px;
+  height: 206px;
+`;
+
+const LocationMain = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  width: 310px;
+  height: 18px;
+  background-color: #ffffff;
+`;
+
+const DescriptionMain = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  margin-top: 22px;
+  width: 310px;
+  height: auto;
+  background-color: #ffffff;
+`;
+
+const Description = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+  width: 60px;
+  height: 18px;
+  font-family: Poppins;
+  font-size: 12px;
+  font-weight: 600;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #229c7f;
+  background-color: #ffffff;
+`;
+
+const DescriptionDescr = styled.div`
```

Comment:
Kakvo je ovo ime komponente? Molio bih deskriptivnije ime. :) Ugl, dosta je slična komponenti Description. Iskoristi stilove koje možeš na  slj način const nekaKomponenta = styled(Description)``;